.changes/unreleased/Docs-20220404-201203.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Docs
+body: Bump jekyll from 3.8.7 to 3.9.0
```

Comment:
I guess we just use this to build the getdbt styles?

Reply:
Yep, that's my understanding.  I wasn't sure if I should include a changelog entry for this or not.  I can remove it if you think it would be confusing.

Reply:
I think it's fine